powerbi-docs/developer/visuals/license-api.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Licensing API
+description: This article discusses how to retrieve the Power BI visual's licenses and trigger the display of out of box license related notifications to the PBI user.
+author: 
+ms.author: 
+ms.reviewer: 
+ms.service: powerbi
+ms.subservice: powerbi-custom-visuals
+ms.topic: how-to
+ms.date: 02/26/2022
+---
+
+
+# Licensing API
+
+The **Licensing API** allows Power BI visual developers to enforce Power BI visual's licensing policies. Use this API to retrieve information about the licenses that are assigned to each Power BI user.
+You can also use the API to trigger license related notifications. The notifications will appear on the Power BI visual to inform the user that they need to purchase missing licenses.
+The visual should not display its own licensing UX. Instead, use one of Power BI predefined supported notifications as detailed below.  
+Learn more about [Power BI Licensing and Translatability support](https://go.microsoft.com/fwlink/?linkid=2186097).
+
+>[!NOTE]
+>The `licensing` API is available from version 4.4. To find out which version you’re using, run the `pbiviz -V` command.
+
+
+## Retrieve visual's service plans that are assigned to the active user
+
+To get service plans assigned, add a call to `getAvailableServicePlans` (available via `IVisualLicenseManager`).    
```

Comment:
These two lines are not clear to me

Reply:
To retrieve the service plan that was assigned to the user, add a call to `getAvailableServicePlans` (available via `IVisualLicenseManager`). For best results, fetch the service plan (license) once, in the `constructor` call, and save the results.